.github/workflows/checks.yml

```diff
@@ -1,25 +1,26 @@
 name: Format and Lint
 
-env:
-  TURBO_TELEMETRY_DISABLED: 1
-
 on:
   pull_request:
   push:
     branches:
       - main
 
+permissions: read-all
+
```

Comment:
Does this need to be `read-all`? `read` is more restrictive unless I'm missing something

Reply:
You're totally right. I misunderstood what read-all does